tests/basic/upgrade_cluster_test.go

```diff
@@ -89,6 +89,18 @@ var _ = Describe("{UpgradeCluster}", func() {
 					log.Warnf("PDB validation skipped. Current Px-Operator version: [%s], minimum required: [%s]. Error: [%v].", opver, PDBValidationMinOpVersion, err)
 				}
 
+				// validate volume quorum during upgrade
```

Comment:
Please make sure this check works for all different distributions

Reply:
This is being validated on vanilla k8s and OCP clusters